web/admin.py

```
@@ -170,12 +207,18 @@ class CustomUserAdmin(BaseUserAdmin):
         "get_enrollment_count",
         "formatted_date_joined",
         "formatted_last_login",
         "rate_limit_status",
     )
-    list_filter = BaseUserAdmin.list_filter + (EmailVerifiedFilter, "date_joined", "last_login")
+    list_filter = BaseUserAdmin.list_filter + (
+        EmailVerifiedFilter,
+        "date_joined",
+        "last_login",
+        "profile__last_email_event",
+    )
```

Ruff (0.8.2)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)